modules/dag-l1/src/main/scala/org/tessellation/dag/l1/Main.scala

```diff
@@ -20,11 +20,12 @@ import org.tessellation.sdk.resources.MkHttpServer
 import org.tessellation.sdk.resources.MkHttpServer.ServerName
 
 import com.monovore.decline.Opts
+import eu.timepit.refined.api.Refined
 
 object Main extends TessellationIOApp[Run]("", "DAG L1 node", version = BuildInfo.version) {
   val opts: Opts[Run] = cli.method.opts
 
-  val kryoRegistrar: Map[Class[_], Int] = dagL1KryoRegistrar ++ dagSharedKryoRegistrar
+  val kryoRegistrar: Map[Class[_], Int Refined _] = dagL1KryoRegistrar ++ dagSharedKryoRegistrar
```

Comment:
How about
`Map[Class[_], DagL1KryoRegistrationId And SharedKryoRegistrationId]`

Reply:
Eventually if you have more registries then you can use

```
AllOf[DagL1KryoRegistrationId, SharedKryoRegistrationId, AnotherKryoRegistrationId]
```

Reply:
For 
`Map[Class[_], DagL1KryoRegistrationId And SharedKryoRegistrationId]`
there is similar problem as for suggestion from @marcinwadon :
`type mismatch;
  scala.collection.immutable.Map[java.lang.Class[_$1], eu.timepit.refined.api.Refined[scala.Int, eu.timepit.refined.boolean.And[eu.timepit.refined.boolean.Not[eu.timepit.refined.numeric.Less[Int(600)]], eu.timepit.refined.boolean.Not[eu.timepit.refined.numeric.Greater[Int(699)]]]]]|scala.collection.IterableOnce[(java.lang.Class[_$1],eu.timepit.refined.boolean.And[eu.timepit.refined.api.Refined[scala.Int, eu.timepit.refined.boolean.And[eu.timepit.refined.boolean.Not[eu.timepit.refined.numeric.Less[Int(800)]], eu.timepit.refined.boolean.Not[eu.timepit.refined.numeric.Greater[Int(899)]]]], eu.timepit.refined.api.Refined[scala.Int, eu.timepit.refined.boolean.And[eu.timepit.refined.boolean.Not[eu.timepit.refined.numeric.Less[Int(600)]], eu.timepit.refined.boolean.Not[eu.timepit.refined.numeric.Greater[Int(699)]]]]])]`

Reply:
@kpudlik `And` will not work, because it requires both constraints to be met and since our ranges are disjoint, it's impossible.

@kriscieslak 
I tried this snipped and it works as expected
```
package org.tessellation.sdk

import eu.timepit.refined.api.Refined
import eu.timepit.refined.auto._
import eu.timepit.refined.boolean.Or
import eu.timepit.refined.numeric.Interval

object playground {

  type Int5xxOr7xx = Int Refined (Interval.Closed[500, 599] Or Interval.Closed[700, 799])

  val x: Int5xxOr7xx = 501
  val y: Int5xxOr7xx = 701
  val z: Int5xxOr7xx = 601 // compile time error

}
```

Reply:
Yes, It should be `Or` because number can't be in 2 disjoint ranges at the same time.